tasks/github_tasks.py

```diff
@@ -188,3 +189,41 @@ def get_token_from_app(_, app_id_env='GITHUB_APP_ID', pkey_env='GITHUB_KEY_B64')
     from .libs.common.github_api import GithubAPI
 
     GithubAPI.get_token_from_app(app_id_env, pkey_env)
+
+
+@task
+def deduce_team_label(_, changed_files=[]):
+    """
+    Print the github team label name if a single team can
+    be deduced from the changed files
+    """
+    def get_team() -> str | None:
+        from codeowners import CodeOwners
+
+        with open('.github/CODEOWNERS') as f:
+            codeowners = CodeOwners(f.read())
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏IIRC this function already exists in the invoke tasks, you can import it !